Forms/Fields/DateOfBirthDropdownField.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DUCK.Forms.Fields
+{
+	public class DateOfBirthDropdownField : AbstractFormField
+	{
+		[SerializeField]
+		private Dropdown dayDropdown;
+		[SerializeField]
+		private Dropdown monthDropdown;
+		[SerializeField]
+		private Dropdown yearDropdown;
+
+		[SerializeField]
+		private Color onValueChangedColor;
+
+		private int defaultDay;
+		private int defaultMonth;
+		private int defaultYear;
+
+		protected override void Awake()
+		{
+			base.Awake();
+
+			defaultDay = dayDropdown.value;
+			defaultMonth = monthDropdown.value;
+			defaultYear = yearDropdown.value;
+
+			dayDropdown.onValueChanged.AddListener(value => { dayDropdown.captionText.color = onValueChangedColor; });
+			monthDropdown.onValueChanged.AddListener(value =>
+			{
+				monthDropdown.captionText.color = onValueChangedColor;
+				HandleValueChanged(value);
+			});
+			yearDropdown.onValueChanged.AddListener(value =>
+			{
+				yearDropdown.captionText.color = onValueChangedColor;
+				HandleValueChanged(value);
+			});
+
+			monthDropdown.AddOptions(DateOfBirthUtils.Months.ToList());
+			dayDropdown.AddOptions(DateOfBirthUtils.Days.Select(day => day.ToString()).ToList());
+			yearDropdown.AddOptions(DateOfBirthUtils.Years.Select(year => year.ToString()).ToList());
+
+			HandleValueChanged(monthDropdown.value);
+		}
+
+		public override object GetValue()
```

Comment:
Is there no way to genericise the parent class so you're not forced to return object?

Reply:
No you can't if its inherited from Monobehaviour

Reply:
Is it worth providing a GetDateOfBirth function in case a caller ever knows it's a DateOfBirthDropdownField and wants to use it in a type-safe way?